docs/sources/dashboards/export-import.md

```diff
@@ -15,7 +15,7 @@ Dashboards are exported in Grafana JSON format, and contain everything you need
 
 The export feature is accessed in the share window which you open by clicking the share button in the dashboard menu.
 
-{{< figure src="/static/img/docs/export/export-modal-8-0.png" max-width="800px" >}}
+{{< docs-imagebox img="/img/docs/export/export-modal.png" max-width="800px" >}}
```

Comment:
image have should have the version number. So, "export-modal-8-0.png" is correct.

Reply:
here it is because i am reverting the docs, since we want to hide the feature with feature toggle for now.

Reply:
```suggestion
{{< figure src="/static/img/docs/export/export-modal.png" max-width="800px" >}}
```